src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.HostProject.cs

```diff
@@ -60,6 +60,8 @@ public ProjectInfo CreateProjectInfoForCurrentState()
                     metadataReferences: _metadataReferences,
                     isSubmission: false,
                     hostObjectType: null);
+
+                return info.WithIsComplete(isComplete: false);
```

Comment:
Oh, I commented on the other PR against stabilization: I would expect this to be true. Right now we have lots of code in the IDE that explicitly checks for `MiscellaneousFilesWorkspace` that should go away.